packages/application-components/src/components/custom-views/custom-views-selector/types.ts

```diff
@@ -0,0 +1,15 @@
+import type { CustomViewData } from '@commercetools-frontend/constants';
+
+export type TCustomViewLocatorCodeLocationDescriptor = {
+  pathname?: string;
+  search?: string;
+};
```

Comment:
Instead of using the type `LocationDescriptor` from `history`, we define our own similar type.